### PR TITLE
Enchance CSS

### LINKED ISSUE #44: Enhance css on inkuire http /query htmls

### PR DESCRIPTION
**Pull request by pikinier20**
closes #44

### CODE REVIEW

**Review by BarkingBad — APPROVED** [review]

File: inkuire-engine/src/main/scala/org/virtuslab/inkuire/engine/http/HttpServer.scala [hunk 1]
```diff
@@ -54,7 +70,32 @@ class HttpServer extends OutputHandler {
               Ok(
                 gson.toJson(fb),
                 `Content-Type`(MediaType.application.json)
-              )
+            )
+          )
+        case GET -> Root / "styles" =>
```
Comment (KacperFKorban) [inline inkuire-engine/src/main/scala/org/virtuslab/inkuire/engine/http/HttpServer.scala:75]: can those be accessed as static files?

File: inkuire-engine/src/main/scala/org/virtuslab/inkuire/engine/http/Templates.scala [hunk 2]
```diff
@@ -1,46 +1,121 @@
...
+            thead(
+              tr(cls := "result-table-header")(
+                th(cls := "left-header-cell", scoped := "col")("#"),
+                th(scoped := "col")("Name"),
+                th(scoped := "col")("Signature"),
+                th(cls := "right-header-cell", scoped := "col")("Localization")
               )
+            ),
+            tbody(
+              for ((res, i) <- results.getMatches.asScala.toList.zipWithIndex)
```
Comment (KacperFKorban) [inline inkuire-engine/src/main/scala/org/virtuslab/inkuire/engine/http/Templates.scala:87]: I know that there was a for before, but can You change it into a `map`?

File: inkuire-engine/src/main/scala/org/virtuslab/inkuire/engine/http/HttpServer.scala (hunk 1 above)
Comment (pikinier20) [reply inkuire-engine/src/main/scala/org/virtuslab/inkuire/engine/http/HttpServer.scala:75]: Ok, right now I found solution that may work on http4s, I will try to change it

File: inkuire-engine/src/main/scala/org/virtuslab/inkuire/engine/http/Templates.scala (hunk 2 above)
Comment (pikinier20) [reply inkuire-engine/src/main/scala/org/virtuslab/inkuire/engine/http/Templates.scala]: done

**Review by KacperFKorban — APPROVED** [review]

**Pull request merged**